includes/admin/tasks/class-wc-payments-task-disputes.php

```diff
@@ -57,30 +121,59 @@ public function get_parent_id() {
 	}
 
 	/**
-	 * Gets the task content.
-	 *
-	 * @return string
-	 */
-	public function get_content() {
-		return '';
-	}
-
-	/**
-	 * Get the additional info.
+	 * Gets the task subtitle.
 	 *
 	 * @return string
 	 */
 	public function get_additional_info() {
-		return __( 'View and respond', 'woocommerce-payments' );
-	}
+		$disputes_due_within_7d = $this->get_disputes_needing_response_within_days( 7 );
+		$disputes_due_within_1d = $this->get_disputes_needing_response_within_days( 1 );
+
+		if ( count( $disputes_due_within_7d ) === 1 ) {
+			$local_timezone    = new \DateTimeZone( wp_timezone_string() );
+			$dispute           = $disputes_due_within_7d[0];
+			$due_by_local_time = ( new \DateTime( $dispute['due_by'] ) )->setTimezone( $local_timezone );
+
+			if ( count( $disputes_due_within_1d ) > 0 ) {
+				return sprintf(
+					/* translators: %s is time, eg: 11:59 PM */
+					__( 'Respond today by %s', 'woocommerce-payments' ),
+					$due_by_local_time->format( 'g:i A' )
+				);
+			}
+
+			$now  = new \DateTime( 'now', $local_timezone );
+			$diff = $now->diff( $due_by_local_time );
+
+			return sprintf(
+				/* translators: %1$s is a date, eg: Jan 1, 2021. %2$s is the number of days left, eg: 2 days. */
+				__( 'By %1$s – %2$s left to respond', 'woocommerce-payments' ),
+				$due_by_local_time->format( 'M j, Y' ),
```

Comment:
should use the `date_format` option.

Reply:
Updated in f4bb038. I'll look at incorporating the store's formatting for the Overview task as well.

Reply:
> I couldn't find a single case where the frontend date is formatted using the store's date formatting, so I'll leave it for now.

Nor could I on a quick review. Seems odd especially seeing the repetition of `M j, Y`. I've created a new issue to track this #6567 as it is not related to this project.